domain/fhir/client.go

```diff
@@ -122,3 +140,19 @@ func resolveResourcePath(resource interface{}) (string, error) {
 	}
 	return resourceType + "/" + resourceID, nil
 }
+
+func buildRequestURI(parts ...string) string {
```

Comment:
you could use `Path.Join`?

Reply:
I assumed it uses the OS-specific path separator, which wouldn't be correct for windows (`\`), but apparentely it always uses forward-slash (`/`), so probably: yes.